src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/NotYetReady/Dex2Hex.txt

```diff
@@ -0,0 +1,124 @@
+﻿// ******** INTEGER PARAMETERS WITHOUT PLACES ********
+
+>> Dec2Hex(0)
```

Comment:
For both tests,  also test  Blank,(), Error() input params.
Often error just propagates.